DearPyGui/src/core/cpp.hint

```diff
@@ -11,12 +11,12 @@ mvAppItemType getType() const override { return x; }\
 std::string getStringType() const override { return std::string(#x); }\
 std::string getParserCommand() const override { return parser; }
 
-#define MV_START_COLOR_CONSTANTS public: static const std::vector<std::tuple<std::string, long, mvColor>>& GetColorConstants()\
-        {static std::vector<std::tuple<std::string, long, mvColor>> constants ={
+#define MV_START_COLOR_CONSTANTS public: static const std::vector<std::tuple<std::string, long, mvColor, mvColor>>& GetColorConstants()\
+        {static std::vector<std::tuple<std::string, long, mvColor, mvColor>> constants ={
 #define MV_END_COLOR_CONSTANTS };return constants;}
 #define MV_START_STYLE_CONSTANTS public: static const std::vector<std::tuple<std::string, long, float, float>>& GetStyleConstants()\
         {static std::vector<std::tuple<std::string, long, float, float>> constants ={
 #define MV_END_STYLE_CONSTANTS };return constants;}
-#define MV_CREATE_CONSTANT_PAIR(x, y) {#x, x, y}
+#define MV_CREATE_CONSTANT_PAIR(x, y) {#x, x, y, y}
```

Comment:
Do we still need MV_CREATE_CONSTANT_PAIR?

Reply:
for any colors we don't want to define the disabled for yes, technically it adds a tuple so maybe we should rename constant pair?